github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -574,9 +574,9 @@ presubmits:
         - --github-token-path=/etc/github/oauth
         - --fix-org=true
         - --fix-org-members=true
-        - --fix-teams=false
-        - --fix-team-members=false
-        - --fix-team-repos=false
```

Comment:
Out of curiosity, why did we have this set to false? AIUI we always need to create teams/members/repos if they are not present?

Reply:
In a previous version of peribolos this presubmit seemed to fail when we set this to true, so I set those flags to false (I figure they changed the `dry-run` logic).